packages/aws-cdk-lib/aws-opensearchservice/README.md

```diff
@@ -533,3 +533,25 @@ const domain = new Domain(this, 'Domain', {
   ipAddressType: IpAddressType.DUAL_STACK,
 });
 ```
+
+## Using Coordinator node with NodeOptions
+
+You can specify coordinator as a valid value for node type.
```

Comment:
Can you please provide a link to related docs?

Reply:
Sure - https://docs.aws.amazon.com/opensearch-service/latest/APIReference/API_NodeOption.html